### ISSUE TITLE
After loading a stage spotlight seems to be stuck on 1 FPS 

### ISSUE THREAD

**Issue by KebabaG** [jupahe64/Spotlight#31]

**Comment by xInspectrEx** [comment]
Yeah, I'm getting the same issue. It was perfectly fine on Windows 10, but ever since I got Windows 11, it's been very laggy

**Comment by antgame11** [comment]
check the discord https://cdn.discordapp.com/attachments/552862949652299856/828932873675210772/Spotlight-CreateViewGroups.7z

**Comment by Ricca665** [comment]
@ant-7802 the link is no longer avaible

**Comment by RugiSerl** [comment]
@Ricca665 I have the same problem too, but thankfully I had a folder containing the spotlight files from an older installation. I have uploaded it on github : https://github.com/RugiSerl/spotlight_release_fix. I think that should work
